v2/api/atlas_client.go

```diff
@@ -12,7 +12,7 @@ const (
 	// CloudURL is default base URL for the services.
 	DefaultCloudURL = "https://cloud.mongodb.com/"
 	// Version the version of the current API client. Should be set to the next version planned to be released.
-	Version    = "2.0.0-dev1"
+	Version    = "1.230201.0-dev1"
```

Comment:
would this be autoupdated? good to do as a follow up tho

Reply:
I'm going to set env var in the generator job and we should see it. Left it intentionally to have the effect in the PR.

Reply:
Actually merging PR with wrong value seems wrong. Will regenerate it now

Reply:
```suggestion
	Version    = "1.20230201.0-dev1"
```